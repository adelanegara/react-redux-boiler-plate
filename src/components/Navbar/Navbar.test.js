import React from "react";
import { screen, render } from "@testing-library/react";
import { NavbarUnwrapped } from ".";

let wrapper;
beforeEach(() => {
  const { getByTestId } = render(<NavbarUnwrapped />);
  wrapper = getByTestId("navbar");
});

test("test navbar", () => {
  expect(wrapper).toBeInTheDocument();
});

test("Test title", () => {
  const wrapper = screen.getByText(/CAR BOOKING/);
  expect(wrapper).toBeInTheDocument();
});

test("Test snapshot", () => {
  const wrapper = screen.getByText(/CAR BOOKING/);
  expect(wrapper).toMatchSnapshot();
});
