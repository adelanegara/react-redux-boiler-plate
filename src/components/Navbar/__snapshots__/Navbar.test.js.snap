// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
<a
  class="navbar-brand"
  href="/home"
>
  CAR BOOKING
</a>
`;
